### PR TITLE
FIX: Add some logging

### PR DESCRIPTION
**Pull request by larsoner**
We should start adding logging statements like these.

### CODE REVIEW

File: mne/preprocessing/maxwell.py [hunk 1]
```diff
@@ -565,8 +570,10 @@ def _overlap_projector(data_int, data_res, delta=0.02):
     C_mat = np.dot(Q_int.T, Q_res)
 
     # Compute angles between subspace and which bases to keep
-    _, S_intersect, Vh_intersect = svd(C_mat)
+    _, S_intersect, Vh_intersect = svd(C_mat, full_matrices=False)
     retain_inds = S_intersect >= (1 - delta)
+    logger.info('    Projecting out %s tSSS components for %s-%s'
+                % ((~retain_inds).sum(), tlims[0], tlims[1]))
```
Comment (larsoner) [inline mne/preprocessing/maxwell.py:576]: This says that 53 and 46 components are rejected, by the way. This seems like way too many...

Comment (larsoner) [inline mne/preprocessing/maxwell.py:576]: (for a sample file I processed)

Comment (wronk) [inline mne/preprocessing/maxwell.py:576]: Ya, that's quite a few... Were you using default delta (for the correlation limit) of 0.02?

Comment (larsoner) [inline mne/preprocessing/maxwell.py:576]:
```
import mne
raw = mne.io.Raw(mne.datasets.testing.data_path() + '/SSS/test_move_anon_raw.fif',
                 proj=False, allow_maxshield=True, preload=True)


def test():
    mne.preprocessing.maxwell_filter(raw, st_dur=10., verbose=True)


if __name__ == '__main__':
    import time
    t0 = time.time()
    test()
    print(time.time() - t0)
```

See if you get the same thing.

Comment (wronk) [inline mne/preprocessing/maxwell.py:576]: I think this is confusion brought about by bad naming. In that code, `retain_inds` refers to the vectors to keep to be projected out. It's not referring to the signal space directions that will be left over after the projection.

I'll update the variable names to be more explicit. There should be 27 and 34 components projected out in your example.

Comment (wronk) [inline mne/preprocessing/maxwell.py:576]: (which still seems like quite a few...)

**Pull request merged**